package/system/ca-certificates/Makefile

```diff
@@ -41,10 +41,17 @@ endef
 
 MAKE_PATH := work
 
+MAKE_INSTALL_FLAGS := \
+	DESTDIR="$(PKG_INSTALL_DIR)" \
+	SBINDIR="/usr/sbin" \
+	CERTSDIR="usr/share/ca-certificates" \
+	LOCALCERTSDIR="usr/local/share/ca-certificates"
+
```

Comment:
Mhhh commit description doesn't mention this change. Can you better describe this and provide better title / commit description?

Reply:
I dropped this commit in favor of another plan for the ca-certificates package which I'm going to handle in a new PR